content/docs/command-reference/remote/modify.md

```diff
@@ -582,6 +582,69 @@ more information.
 
 </details>
 
+<details>
+
+### Click for WebDAV
+
+- `token` - token for WebDAV server, can be empty in case of using
+  `user/password` authentication.
+
+  ```dvc
+  $ dvc remote modify --local myremote token mytoken
+  ```
+
+- `user` - username for WebDAV server, can be empty in case of using `token`
+  authentication. The order in which DVC searches for username is:
+
+  1. `user` specified in one of the DVC configs;
```

Comment:
What do we mean by "DVC configs" here?

Reply:
This whole order explanation could probably be after the example.

Reply:
I don't know, took this from the `HTTP` section as it behaves the same. Probably it means one of the config files at _system_, _global_, _project_ or _local_ level?

Should this be made more clear? At least `HTTP` and `SSH` have similar sections. Is there already an explanation on configs somewhere which could be linked? Or is it enough here to note that explicit `user` config takes precedence over `user` extracted from `url`?

Reply:
Ah yeah thanks for mentioning that! Those need to be rewritten for sure. I did all the changes for the current text in c9df0b1b. We can follow that lead here 🙂